iam-policy-tls-cert-restore-s3.tf

```diff
@@ -0,0 +1,31 @@
+resource "aws_iam_policy" "tls_cert_restore_s3" {
+  provider = aws.clientaccount
+  for_each = aws_route53_zone.clusters
+  name     = "tls-cert-restore-s3-${aws_route53_zone.clusters[each.key].name}"
+  policy   = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "s3:List*",
+        "s3:ListBucket*"
+      ],
+      "Resource": [
+        "${module.common_s3.primary_s3_bucket_arn}"
+      ]
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "s3:GetObject*"
+      ],
+      "Resource": [
+        "${module.common_s3.primary_s3_bucket_arn}/*"
```

Comment:
Can you trim this down and make it similar to the other IAM permissions so that it only has access to it's respective cluster.     "${module.common_s3.primary_s3_bucket_arn}/${aws_route53_zone.clusters[each.key].name}/tls-cert-backups/*"